### ISSUE TITLE
“Select your county” button has an insufficient accessible name and lacks a role of button [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#231]
The “select your county” button found on the “NY COVID Data” screen is labeled by whichever county is currently selected (e.g., “Albany”). Without the visual information provided by the “filter” icon and the surrounding content, TalkBack users are unlikely to understand the purpose of the button without activating it and exploring the next screen. The button also does not convey a role of button.

![data Tab - Menu control - All counties in NY](https://user-images.githubusercontent.com/7034772/92004261-00fc0f00-ed10-11ea-9a23-eda6df2fcb26.png)

Severity: High

**Comment by floridemai** [comment]
@jonnyjames Should be addressed in build 34 and after.

**Comment by jonnyjames** [comment]
@floridemai This is now fixed. Closing this issue.
